src/AbstractProxyFixer.php

```diff
@@ -23,47 +24,71 @@
 abstract class AbstractProxyFixer extends AbstractFixer
 {
     /**
-     * @var FixerInterface
+     * @var array<string, FixerInterface>
      */
-    protected $proxyFixer;
+    protected $proxyFixers;
 
     public function __construct()
     {
-        parent::__construct();
+        foreach (Utils::sortFixers($this->createProxyFixers()) as $proxyFixer) {
+            $this->proxyFixers[$proxyFixer->getName()] = $proxyFixer;
+        }
 
-        $this->proxyFixer = $this->createProxyFixer();
+        parent::__construct();
```

Comment:
btw, why constructor was moved?

Reply:
Because parent constructor initializes the configuration with default values by calling `configure([])`. If the `configure` methods needs to access a proxied fixer, the proxied fixers must be available. See [#2619](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2619/files#diff-60306f9e47866c05bdcbedf9aa5dc03dR77) for an example.